package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>moveit_middleware_benchmark</name>
+  <version>0.0.0</version>
+  <description>Middleware Benchmark Tool For MoveIt Pipelines</description>
```

Comment:
```suggestion
  <description>Middleware Benchmark Tool For MoveIt 2</description>
```

Reply:
Fixed in https://github.com/CihatAltiparmak/moveit_middleware_benchmark/pull/5/commits/fa4710c5de5217be352e51797c365f8fc1e1ae1f commit